### PR TITLE
Update description and repo in package.json

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-cli-deploy-redis",
   "version": "0.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "description": "Ember CLI Deploy plugin to deploy index.html into redis.",
```
Comment (achambers) [inline package.json]: @lukemelia I'm keen to not be specific about index.html as this plugin could be used to upload anything to redis.  Thoughts?

Comment (achambers) [inline package.json]: @lukemelia What are your thoughts on this?

**Comment by lukemelia** [comment]
@achambers agreed. updated

**Comment by achambers** [comment]
Cheers

**Pull request merged**